types/express-brute-mongo/tslint.json

```diff
@@ -0,0 +1,8 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        // TODO
```

Comment:
Why `TODO`?

Reply:
It would be nice to eventually get rid of the `Object` type (by documenting the options), and supply a version for the library in the header. Then these lint rules wouldn't need to be disabled.